hr/overview/employee_form.rst

```diff
@@ -0,0 +1,123 @@
+=============
+Employee Form
+=============
+
+
+Create an employee Form
+=======================
+
+You can **Create** an employee from the employee dashboard or directly from a department.
+
+
+.. image:: media/create-employee.png
+   :align: center
+   :alt: How to create an employee
+
+If you have an existing list of employees, you can *Import* them in a CSV or Excel file, to batch import your employees,
+simply click on the *import* button on the dashboard.
+
+.. note::
+   Need help ? `how to import your contacts <https://www.odoo.com/slides/slide/contacts-and-importing-data-609?fullscreen=1>`_.
+
+Complete employee form
+======================
+
+The employee form allows you to add tags, a picture and edit all the general information about your employee.
+You can also create and edit departments and a manager for your employee.
+
+
+.. image:: media/employee-form.png
+   :align: center
+   :alt: Empty employee form
+
+Print
+=====
+
+From the employee form, *Print Badge*, *Time Off Summaries* and *Individual Accounts* for each employee.
```

Comment:
```suggestion
From the employee form, *Print Badge*, *Time Off Summaries* and *Individual Accounts* for each employee.
```

Maybe introduce a quick reason/motivation for why this is usefu + a quick explanation about the options (as the Print Bagde and Individual Accounts are not very intuitive, I feel (: )?